stories/apps/launch/AgaveWordCount.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import AppLaunchWizard from "../../../src/components/apps/launch/AppLaunchWizard";
+
+import AgaveWordCountApp from "./data/AgaveWordCount";
+
+import {
+    ANALYSIS_OUTPUT_DIR,
+    STARTING_PATH,
+    initMockAxiosFileFolderSelector,
+    submitAnalysis,
+} from "./constants";
+
+export const AgaveWordCount = () => {
+    initMockAxiosFileFolderSelector();
+
+    return (
+        <AppLaunchWizard
+            notify={false}
+            defaultOutputDir={ANALYSIS_OUTPUT_DIR}
+            startingPath={STARTING_PATH}
+            submitAnalysis={submitAnalysis}
+            app={AgaveWordCountApp}
+        />
+    );
+};
+
+export default { title: "Apps / Launch" };
```

Comment:
Yep. This message is currently displayed in a popup dialog when a user launches an HPC app in the current DE, and since popups are really obtrusive and can get annoying fast, we give the user a preference to no longer show the popup in the future.

I think I like this better, where the message is always displayed, but not in an obtrusive popup. So hopefully it's easy to get past but not easily forgotten 👍